File: javascript/Fashion/wallet.js
basic.forever(() => {
    if (input.lightLevel() > 16) {
        basic.showLeds(`
            # # . # #
            # # . # #
            . # # # .
            . # . # .
            . # . # .
            `)
        basic.showLeds(`
            . . # . .
            . . # . .
            # . . . #
            # . # . #
            # . # . #
            `)
    } else {        
        // clear screen and wait
        basic.clearScreen()
        basic.pause(3000)
    }
})

File: haskell/Fashion/wallet.hs
import MicroBit
import Prelude hiding (map) 

if_b :: MicroBit
if_b = (showLED $ return 
        [". # . # .",
        "# # # # #",
        "# # # # #",
        ". # # # .",
        ". . # . ."])
        >>
        (showLED $ return 
           [   ". # . # .",
        "# # # # #",
        "# # # # #",
        ". # # # .",
        ". . # . ."]
        )

else_b :: MicroBit
else_b = clear >> pause 3000


led :: MicroBit
led = 
    assert (lightLevel .> 0)
    (if lightLevel .>16 then if_b else else_b) 
